openlibrary/views/loanstats.py

```diff
@@ -158,19 +158,24 @@ def get_cached_activity_stream(limit):
 class activity_stream(app.view):
     path = "/trending(/?.*)"
 
-    def GET(self, page=''):
-        if not page:
+    def GET(self, mode=''):
+        page = int(web.input(page=1))
```

Comment:
```suggestion
        i = web.input(page=1)
        page = i.page
```
Looks like `Storage` objects cannot be cast to integers (as per https://testing.openlibrary.org/trending/now?debug=true).

Tested the above locally.